Align RabbitMQ installation guide (#5063)
magento/devdocs: guides/v2.3/install-gde/prereq/install-rabbitmq.md

* Starting and stopping the broker
* System limits

> ## Install Magento with RabbitMQ connection to {{site.data.var.ce}} or {{site.data.var.ee}} 

Reviewer: ```suggestion
## Install Magento with RabbitMQ and connect to {{site.data.var.ce}} or {{site.data.var.ee}} 
```

> If you installing Magento after you installed RabbitMQ, add the following command line parameters when you install {{site.data.var.ce}} or {{site.data.var.ee}}:

Reviewer: ```suggestion
If you installed Magento after you installed RabbitMQ, add the following command line parameters when you install {{site.data.var.ce}} or {{site.data.var.ee}}:
```

|`--amqp-virtualhost`|The virtual host for connecting to RabbitMQ. The default is `/`.
|`--amqp-ssl`|Indicates whether to connect to RabbitMQ. The default is `false`. If you set the value to true, see Configure SSL for more information.|

## Connect RabbitMQ to {{site.data.var.ce}} or {{site.data.var.ee}} 

> If you already had Magento installed and you want to connect it with RabbitMq, add `queue` section in the `<install_directory>/app/etc/env.php` file so that it is similar to the following:

Reviewer: ```suggestion
If you already had Magento installed and you want to connect it to RabbitMQ, add a `queue` section in the `<install_directory>/app/etc/env.php` file so that it is similar to the following:
```

```php
'queue' =>
  array (
    'amqp' =>
    array (
      'host' => 'rabbitmq.example.com',
      'port' => '11213',
      'user' => 'magento',
      'password' => 'magento',
      'virtualhost' => '/'
     ),
  ),
```

> After, please run `bin/magento setup:upgrade` for apply changes and create required queues in RabbitMQ

Reviewer: ```suggestion
Then, run `bin/magento setup:upgrade` to apply the changes and create the required queues in RabbitMQ.
```
